prov/efa/src/rdm/efa_rdm_ep_utils.c

```diff
@@ -595,3 +596,80 @@ ssize_t efa_rdm_ep_trigger_handshake(struct efa_rdm_ep *ep,
 
 	return 0;
 }
+
```

Comment:
why put this function in ep_utils.c rather than efa_rdm_pke_nonreq.c?

Reply:
This function operates on an endpoint, therefore need to be in ep,